mobile/src/components/WalkEnrollments/index.js

```diff
@@ -47,8 +50,10 @@ function WalkInEnrollmentRouteCheck({pageName}) {
                 return <SelectComorbidity/>;
             case FORM_WALK_IN_VERIFY_MOBILE:
                 return <VerifyMobile/>;
+            case FORM_WALK_IN_MOSIP_ENROLL_FORM:
+                return <MosipAuth />;
             case FORM_WALK_IN_VERIFY_OTP:
-                return <VerifyOTP/>;
+                return <VerifyOTP state={state} />;
```

Comment:
We should be able to get the context and state variables inside the `VerifyOTP` component instead of passing it as a prop.

Reply:
ok